lib/Model.ts

```diff
@@ -1215,7 +859,7 @@ query('id').eq(1).where('name').begienwith('foxy')
 Model.query = function (NewModel, query, options, next) {
   if (typeof options === 'function') {
     next = options;
-    options = null;
+    options = undefined;
```

Comment:
Why are we switching this to undefined? It's done in a few other places also.

Reply:
undefined over null is a tsconfig setting i've always used, it's mostly my preference as undefined is the default and null is an assigned value.